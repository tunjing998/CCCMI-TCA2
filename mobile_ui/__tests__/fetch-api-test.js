import React, {Component} from 'react';
import {View, Text} from 'react-native';
import fetchApi from '../screens/fetchApi';
import renderer from 'react-test-renderer';

test('fetchApi snapShot', () => {
  const snap = renderer.create(<fetchApi />).toJSON();
  expect(snap).toMatchSnapshot();
});
