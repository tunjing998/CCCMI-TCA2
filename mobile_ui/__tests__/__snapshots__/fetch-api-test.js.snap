// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fetchApi snapShot 1`] = `<fetchApi />`;
